ghc-lib-gen/src/Main.hs

```diff
@@ -148,25 +151,31 @@ calcParserModules = do
         , "-dep-makefile .parser-depends"
         , "-M"]
         ++ include_dirs
-        ++ ["-ignore-package ghc-lib-parser -package ghc", "-package base"]
-        ++ hs_source_dirs ++ ["-ighc-lib/stage0/libraries/ghc-heap/build"]
+        ++ ["-ignore-package ghc-lib-parser"
+           , "-ignore-package ghc"
+           -- , "-ignore-package ghci"
```

Comment:
If we don’t need that line, let’s remove it.

Reply:
Actually we do but the ghc-lib-parser line we don't. This program is so fiddly to get right (and I'm sure this isn't the end of it...)!